index.php

```diff
@@ -12,6 +12,7 @@ function getTool(
     <div class="col col-sm-4">
         <div style="cursor:pointer"
         class="info-box ' . $bgclass . '"
+        id="' . str_replace(' ', '', strtolower($displayname)) . '"
```

Comment:
i changed this to be lowercase and without spaces.

Reply:
Great Idea!